rag/rag_practice/yes24_bestseller_crawling.py

```
@@ -0,0 +1,100 @@
+                continue
+            tags.append(txt.lstrip('#'))
+    tags_map[link] = tags
+
+print("수집한 상세 링크 수:", len(book_urls))
+print("예시 태그:", tags_map.get(book_urls[0], []))
+
+
+dic = []
+for index, book_url in enumerate(book_urls, start=1):
+    try:
+        html = urlopen(book_url)
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)